lib/GenomeFileUtil/core/GenomeUtils.py

```diff
@@ -55,17 +55,24 @@
                     "designate trans_splicing.",
     "premature_stop_codon": "Extra in frame stop codon found.",
     "mRNA_fail_parent_coordinate_validation": "This mRNA lists CDS {} as its "
-                    "corresponding CDS, but fails coordinate validation.",
+                    "corresponding CDS, but fails it coordinate validation.",
     "CDS_fail_child_of_mRNA_coordinate_validation": "This CDS lists mRNA {} as its "
-                    "corresponding mRNA, but fails coordinate validation.",
+                    "corresponding mRNA, but fails it coordinate validation.",
     "CDS_fail_child_of_gene_coordinate_validation": "This CDS lists gene {} as its "
-                    "corresponding gene, but fails coordinate validation.",
+                    "corresponding gene, but fails it coordinate validation.",
     "genes_mRNA_child_fails_location_validation": "The mRNA {} lists this gene as its "
-                    "corresponding parent gene, but fails coordinate validation.",
+                    "corresponding parent gene, but it fails coordinate validation.",
     "genes_CDS_child_fails_location_validation": "The CDS {} lists this gene as a "
-                    "corresponding parent gene, but fails coordinate validation.",
+                    "corresponding parent gene, but it fails coordinate validation.",
     "mRNAs_parent_gene_fails_location_validation": "This mRNA lists gene {} as its "
-                    "corresponding parent gene, but fails coordinate validation."
+                    "corresponding parent gene, but it fails coordinate validation.",
+    "generic_parents_child_fails_location_validation": "This feature lists feature {} as its "
```

Comment:
Might be cleaner to just use generic messages for all the types but I leave that to you.